scripts/publish.js

```diff
@@ -103,6 +106,31 @@ module.exports = (testing = '') => new Promise((resolve, reject) => {
     });
 });
 
+function webhook(message) {
+    const CIRCLE_PROJECT_REPONAME = process.env.CIRCLE_PROJECT_REPONAME;
+
+    if (!CIRCLE_PROJECT_REPONAME) {
+        return;
+    }
+
+    return webhookNotifier(
+        {
+            attachments: [
+                {
+                    fallback: `${CIRCLE_PROJECT_REPONAME} package was automatically published`,
```

Comment:
how about adding the dev name somewhere here? is it even possible? since we publish from a generic user.

adding the dev somewhere along the publication process will help to debug/retro a pressing issue etc.

Reply:
![image](https://user-images.githubusercontent.com/516342/34196229-d5ea9ac6-e569-11e7-96fa-1424c97f4834.png)